br/pkg/lightning/backend/local/local.go

```diff
@@ -517,7 +517,7 @@ func NewBackend(
 		// If the time too short, we may scatter a region many times, because
 		// the interface `ScatterRegions` may time out.
 		pd.WithCustomTimeoutOption(60*time.Second),
-		pd.WithMaxErrorRetry(3),
+		pd.WithMaxErrorRetry(100),
```

Comment:
100 is the default value, I prefer we search and remove all `WithMaxErrorRetry`

Reply:
Removing all of these will also affect the import by lightning process. If there is really something wrong with PD (like #48209), the error will be reported after at least 100 seconds, is that expected?